libs/core/src/lib/action-sheet/action-sheet-control/action-sheet-control.component.ts

```diff
@@ -36,4 +37,16 @@ export class ActionSheetControlComponent {
     onClick(): void {
         this.clicked.emit();
     }
+
+    /** @hidden */
+    _focus(): void {
+        const elemChild = this._elementRef.nativeElement.firstChild;
+        if (elemChild && elemChild.getAttribute('tabindex') !== '-1') {
```

Comment:
it will return `null` for buttons and other elements that are focusable by default and don't need `tabindex` to be explicitly added.